feconf.py

```diff
@@ -1008,3 +1009,11 @@ def get_empty_ratings():
 
     }
 }
+
+# A dict representing available landing pages, having subject as a key and list
+# of topics as the value.
+# Note: This dict need to be keep in sync with frontend TOPIC_LANDING_PAGE_DATA
```

Comment:
need --> needs. Also specify what the frontend file actually is.

Add a similar comment in the frontend file as well, referring back to here.

Reply:
Done!